index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="styles.css"></link>
+<meta charcode="UTF-8">
+<title>Survey Form
+</title>
```

Comment:
I suggest you throw more light on this, kindly make sure your comments are very descriptive and easy to understand😉. don't forget we learnt in our lesson on [how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) that we should make our comments descriptive easy to understand 😉.  don't forget, it may be clear to you, but not clear to the one reading it😉.